worker/mozillaGradingWorker/ciphersuites.go

```diff
@@ -0,0 +1,1295 @@
+package mozillaGradingWorker
+
+//OpenSSLCiphersuites contains all the ciphersuites recognised by OpenSSL
+var OpenSSLCiphersuites = `{
```

Comment:
I generated the file by running the OpenSSL output through Go, so it is even more silly that I did not include it :no_mouth: